### ISSUE TITLE
Early stop on train

### ISSUE THREAD

**Issue by lbugnon** [experiencor/keras-yolo3#327]
Hi, If i get it right, early stop is used on train (in fact, reported loss is from train set). Is there any reason for that? I mean, why validation set is used only as a test set?

**Comment by pkienle** [comment]
I would also be interested if this was an intentional choice to not monitor val_loss and if yes, because of which reasons.
